test/integration/01-setup/01-funds-bitcoind-lnds.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { baseLogger } from "src/logger"
+import { btc2sat } from "src/utils"
+import { MainBook } from "src/mongodb"
+import { liabilitiesReserve, lndAccountingPath } from "src/ledger/ledger"
+import {
+  checkIsBalanced,
+  lnd1,
+  lndOutside1,
+  bitcoindClient,
+  getChainBalance,
+  fundLnd,
+  waitUntilSyncAll,
+} from "test/helpers"
+
+jest.mock("src/realtimePrice", () => require("test/mocks/realtimePrice"))
+
+const defaultWallet = ""
+
+beforeEach(async () => {
+  await waitUntilSyncAll()
+})
+
+describe("Bitcoind", () => {
+  it("should be funded mining 10 blocks", async () => {
+    const numOfBlock = 10
+    try {
+      const { name } = await bitcoindClient.createWallet(defaultWallet)
+      // depends of bitcoind version. needed in < 0.20 but failed in 0.21?
+      expect(name).toBe(defaultWallet)
+    } catch (error) {
+      baseLogger.warn({ error }, "bitcoind wallet already exists")
+    }
+
+    const bitcoindAddress = await bitcoindClient.getNewAddress()
+    await bitcoindClient.mineAndConfirm(numOfBlock, bitcoindAddress)
+    const balance = await bitcoindClient.getBalance()
+
+    // invalid only when there are problems deleting channels and
+    // they get confirmed while mineAndConfirm (101 blocks)
```

Comment:
not sure to fully understand the tests/comments here

Reply:
didn't deleted after remove block reward removal, just removed https://github.com/GaloyMoney/galoy/pull/340/commits/7eba5e062a25234804eac0b372da682cff2a8ce2